lib/cjs/src/Admin/admin-pages/SideBar.d.ts

```diff
@@ -0,0 +1,5 @@
+interface SideBar {
+    elt: any;
```

Comment:
What is `elt`? I cannot tell by the variable name or the type. Could be something I am not familiar with?